RebelAllianceBank/Users/Customer.cs

```diff
@@ -263,6 +263,85 @@ public void Transfer()
         }
+        
+        public void Deposit()
+        {
+            if (_bankAccounts.Count < 1)
+            {
+                Console.WriteLine($"{TextColor.Red}Du har inga konton att sätta in pengar på {TextColor.NORMAL}. " +
+                                  $"\n\nTryck enter för att fortsätta");
+                while (Console.ReadKey(true).Key != ConsoleKey.Enter) { };
+                return;
+            }
+            var menu = new SelectOneOrMore(["id", "Konto Namn", "Saldo", "Valuta"], PopulateAccountDetails(_bankAccounts));
+
+            Console.Clear();
+            Markdown.Paragraph($"{TextColor.Yellow}Till{TextColor.NORMAL} vilket konto vill du sätta in dina pengar? ");
+            int[] accountToIndex;
+
+            while ((accountToIndex = menu.Show()).Length == 0)
+            {
+                Console.Clear();
+                Markdown.Paragraph($"{TextColor.Red}Välj ett alternativ{TextColor.NORMAL}");
+            }
+
+            Console.Clear();
+
+            var accountTo = _bankAccounts[accountToIndex[0]];
+            
+            List<IBankAccount> updatedAccounts = [
+                accountTo,
+            ];
+
+            Console.Clear();
+
+            // Header
+            decimal moneyToDepositinAccountCurrency = 0; 
+            bool runLoopSetAmount = true;
+            while (runLoopSetAmount)
+            {
+                string defaultcurrency = "SEK";
+                Markdown.Header(HeaderLevel.Header2, $"Hur mycket i {defaultcurrency} vill du sätta in på " +
+                                                     $"{accountTo.AccountName}?\n");
+                Markdown.Table(["id", "Konto Namn", "Saldo", "Valuta"], PopulateAccountDetails(updatedAccounts));
+                Console.Write("\nBelopp: ");
+
+                decimal moneyToDeposit;
+                while (!decimal.TryParse(Console.ReadLine(), out moneyToDeposit) || moneyToDeposit < 0)
+                {
+                    Markdown.Paragraph($"Välj ett nytt belopp som är större än 0");
+                }
+
+                moneyToDepositinAccountCurrency = moneyToDeposit *
+                                                          Bank.exchangeRate.CalculateExchangeRate(defaultcurrency,
+                                                              accountTo.AccountCurrency);
+                string answer = "";
+                while (answer != "ja" && answer != "j" && answer != "nej" && answer != "n")
+                {
+                    Console.WriteLine($"Du har angett att du vill sätta in {moneyToDeposit} {defaultcurrency} " +
+                                      $"({moneyToDepositinAccountCurrency:N2} {accountTo.AccountCurrency})\n" +
+                                      $"\n" +
+                                      $"Stämmer detta?");
+                    answer = Console.ReadLine().ToLower();
+                    if (answer != "ja" && answer != "j" && answer == "nej" && answer != "n")
```

Comment:
is it supposed to say "Ogiltigt val!Tryck enter för att lägga in ett nytt belopp!" when typing "nej"
I think you forgot to write `answer != "nej"`

Reply:
Ja! Tack! :)